{{cookiecutter.repo_name}}/config/celery_config.py

```diff
@@ -0,0 +1,4 @@
+# This tells Celery which Django settings module it should use.
+# This decouples the taskapp configuration from a specific instance of the project.
+
+DJANGO_SETTINGS_MODULE = 'config.settings.local'
```

Comment:
Is this so that the celery tasks can run under different settings modules? In other words, we could change this value to 
`DJANGO_SETTINGS_MODULE = 'config.settings.celery_settings'`?

If that's the case, I'm not certain why this extra module is needed. Is it because of the inherent magic of Celery?